docs/docs-graphql/schema.graphqls

```diff
@@ -585,6 +627,7 @@ type Container {
   withMountedCache(
     """Identifier of the cache volume to mount."""
     cache: CacheVolumeID!
+    expand: Boolean = false
```

Comment:
`withMountedCache` needs an `ArgDoc` so we get the docs here as well.

Reply:
Thank you very much for catching this 👍. fixed.